selfdrive/car/toyota/toyotacan.py

```diff
@@ -38,6 +38,7 @@ def create_accel_command(packer, accel, pcm_cancel, standstill_req, lead, acc_ty
     "RELEASE_STANDSTILL": not standstill_req,
     "CANCEL_REQ": pcm_cancel,
     "ALLOW_LONG_PRESS": 1,
+    "ACC_CUT_IN": fcw_alert,
```

Comment:
should we rename the signal?

Reply:
It's used as cut-in warning in the stock system, it's similar to fcw on both the dashboard and HUD, only difference is it's yellow instead of red.

Reply:
`CUT_IN_WARNING`?

Reply:
![image](https://github.com/commaai/openpilot/assets/12470297/1b84c539-1739-42fe-ba11-8cf3ad4a96fc)

In TS it's being named this